usr/share/rear/layout/prepare/GNU/Linux/100_include_partition_code.sh

```diff
@@ -57,7 +58,8 @@ sync
 
 EOF
 
-    create_partitions "$disk" "$label"
+    # $junk can contain useful DASD-specific fields
```

Comment:
When a variable is named 'junk'
but (sometimes) it contains useful data,
then the varaiable should not be named 'junk'
(even if in most cases that data is useless).

Reply:
ok, I am going to rename it to "extrafields" or something similar where applicable.